definitions/scenarios/upgrade_to_foreman_3_3.rb

```diff
@@ -0,0 +1,92 @@
+module Scenarios::Foreman_3_3
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_scenario
+        confine do
+          feature(:only_foreman) &&
+            (feature(:foreman_server).current_minor_version == '3.2' || \
+            ForemanMaintain.upgrade_in_progress == '3.3')
+        end
+        instance_eval(&block)
+      end
+    end
+
+    def target_version
+      '3.3'
+    end
+  end
+
+  class PreUpgradeCheck < Abstract
+    upgrade_metadata do
+      description 'Checks before upgrading to Foreman 3.3'
+      tags :pre_upgrade_checks
+      run_strategy :fail_slow
+    end
+
+    def compose
+      add_steps(find_checks(:default))
+      add_steps(find_checks(:pre_upgrade))
+    end
+  end
+
+  class PreMigrations < Abstract
+    upgrade_metadata do
+      description 'Procedures before migrating to Foreman 3.3'
```

Comment:
```suggestion
      description 'Procedures before upgrading to Foreman 3.3'
```

Since we cannot change the internal step names, we could at least align our nomenclature that user's see.

Reply:
Yeah we can't maybe while refactoring the whole upgrade scenarios this can be address, forked the issue for same : https://projects.theforeman.org/issues/35252